fix: add callout to mention env variable takes precedence over Admin settings (#1394)
appsmithorg/appsmith-docs: website/docs/getting-started/setup/instance-configuration/README.mdx

  <figcaption align = "center"><i>Access admin settings on the left panel under workspaces</i></figcaption>
</figure>

:::info

> When configuring an Appsmith instance, it's important to note that if you choose to set configurations using [environment variables](#environment-variables), those values take precedence over any values set in the Admin Settings.

Reviewer: ```suggestion
If you have set values using [environment variables](#environment-variables) for your instance, those values take precedence over values specified in the Admin Settings UI.
```
